tests/test_json.py

```diff
@@ -121,6 +121,57 @@ class Squid(proto.Message):
     )
     assert json1 == '{"name":"Steve"}'
 
+    json1 = (
+        Squid.to_json(s, always_print_fields_with_no_presence=False)
+        .replace(" ", "")
+        .replace("\n", "")
+    )
+    assert json1 == '{"name":"Steve"}'
+
+    json1 = (
+        Squid.to_json(
+            s,
+            including_default_value_fields=False,
```

Comment:
I would add two additional cases: when only `including_default_value_fields` is set to `False`, and when only it is set to `True` (i.e. it is the only parameter specified).

(and ditto in test for `to_dict`)

Reply:
I believe the `including_default_value_fields=False` and `always_print_fields_with_no_presence=False` test cases are complete. I added more test cases for `including_default_value_fields=True` and `always_print_fields_with_no_presence=True`

https://github.com/googleapis/proto-plus-python/pull/433/commits/78628035e7789eb12f61cf31b568dc8dc1e2f57f

Reply:
OK, looks good. Thanks for adding these test cases!